src/app/people/people.component.html

```diff
@@ -45,72 +41,79 @@ <h3>You may bring an unlimited number of mentors.</h3>
                 </p>
             </div>
         </div>
-        <div class="half unit row">
-            <fieldset>
-                <input type="button" value="Add Mentor." (click)="addMentor()" [disabled]="!mentorDetailsCompleted">
-            </fieldset>
+        <div class="two unit row">
+            <div class="one half unit column">
+                <button (click)="addMentor()" [disabled]="!mentorDetailsCompleted">Add Mentor.</button>
+            </div>
         </div>
-        <div class="half unit row" *ngFor="let mentor of mentors; let i = index">
-            <fieldset>
+        <fieldset class="one unit persistent row content" *ngFor="let mentor of mentors; let i = index">
```

Comment:
If you add multiple mentors the list of mentors gets extremely squished up, can we add some margin between each mentor?